tests/integration/kubernetes/confidential/k8s-confidential.bats

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bats
+# Copyright 2022-2023 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+IMAGE_REPO="ghcr.io/confidential-containers/test-container"
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	( [ "${KATA_HYPERVISOR}" == "firecracker" ] || [ "${KATA_HYPERVISOR}" == "dragonball" ] || [ "${KATA_HYPERVISOR}" == "qemu-tdx" ] ) && skip "Test not supported yet."
```

Comment:
Same comment I made in this review about having a list and checking the list.

Reply:
Changed.